app/notification_test.go

```diff
@@ -975,11 +971,10 @@ func TestGetMentionKeywords(t *testing.T) {
 
 	profiles = map[string]*model.User{user2.Id: user2}
 	mentions = th.App.getMentionKeywordsInChannel(profiles, true, channelMemberNotifyPropsMap2Off)
-	if len(mentions) != 2 {
-		t.Fatal("should've returned two mention keyword")
-	} else if ids, ok := mentions["First"]; !ok || ids[0] != user2.Id {
-		t.Fatal("should've returned mention key of First")
-	}
+	require.False(t, len(mentions) != 2, "should've returned two mention keyword")
```

Comment:
I would prefer if we did this here
```suggestion
	require.Len(t, mentions, 2, "should've returned two mention keyword")
```